README.md

```diff
@@ -68,11 +68,13 @@ It's possible to specify the Dockerfile to use by:
 steps:
   - command: 'echo wow'
     plugins:
-      - seek-oss/docker-ecr-cache#v1.1.5:
+      - seek-oss/docker-ecr-cache#v1.1.6:
           dockerfile: my-dockerfile
       - docker#v3.0.1
 ```
 
+The subdirectory containing the Dockerfile is the path used for the build's context.
```

Comment:
We can probably get away with the versioning by calling it previously-undefined behaviour 😄